packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -246,34 +259,37 @@ contract('FederatedAttestations', (accounts: string[]) => {
       const issuer1Attestations: AttestationTestCase[] = [
         {
           account: account1,
-          issuedOn: nowUnixTime,
           signer: signer1,
+          issuedOn: nowUnixTime,
         },
         // Same issuer as [0], different account
         {
           account: account2,
-          issuedOn: nowUnixTime,
           signer: signer1,
+          issuedOn: nowUnixTime,
         },
       ]
       const issuer2Attestations: AttestationTestCase[] = [
         // Same account as issuer1Attestations[0], different issuer
         {
           account: account1,
-          issuedOn: nowUnixTime,
           signer: issuer2Signer,
+          issuedOn: nowUnixTime,
         },
         // Different account and signer
         {
           account: account2,
-          issuedOn: nowUnixTime,
           signer: issuer2Signer2,
+          issuedOn: nowUnixTime,
         },
       ]
 
+      let expectedPublishedOn
+
       beforeEach(async () => {
         // Require consistent order for test cases
         await accountsInstance.createAccount({ from: issuer2 })
+        expectedPublishedOn = Math.floor(Date.now() / 1000)
```

Comment:
nit: nervous this may still cause some issues with flakiness. Consider:
- setting `expectedPublishedOn` after the attestations are registered
- then checking that the `actualPublishedOns` are: `issuedOn` <= `actualPublishedOn` <= `expectedPublishedOn` (+ 10 or whatever other additional wiggle room you want to add in)
Wdyt?